crates/bench/src/schemas.rs

```diff
@@ -0,0 +1,332 @@
+use spacetimedb_lib::sats;
+use std::fmt::Debug;
+use std::hash::Hash;
+
+pub const BENCH_PKEY_INDEX: u32 = 0;
+
+// the following piece of code must remain synced with `modules/bencmarks/src/lib.rs`
+// unfortunately, you can't just copy it.
+
+// ---------- SYNCED CODE ----------
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Person {
+    id: u32,
+    name: String,
+    age: u64,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct Location {
+    id: u32,
+    x: u64,
+    y: u64,
+}
+// ---------- / SYNCED CODE ----------
+
+/// Note: the first field will be used as the primary key, when using
```

Comment:
I can't understand how this comment relates to this trait. The first field of what?

Reply:
Of the product type / value conversion of the underlying struct, I'll move it.